databricks/koalas/frame.py

```diff
@@ -9122,6 +9122,178 @@ def gen_new_column_labels_entry(column_labels_entry):
         else:
             return DataFrame(internal)
 
+    def rename_axis(
+        self,
+        mapper: Optional[Any] = None,
+        index: Optional[Any] = None,
+        columns: Optional[Any] = None,
+        axis: Optional[Union[int, str]] = 0,
+        inplace: Optional[bool] = False,
+    ):
```

Comment:
Shall we add the return type annotation as well?

Reply:
Yes, but we need to use `return None` in case of `inplace=True` in order to pass mypy checks.

Reply:
Then maybe we should use `Union["DataFrame", None]` ?

Reply:
According to [mypy docs](https://mypy.readthedocs.io/en/stable/getting_started.html#the-typing-module) : `Optional[str] is just a shorthand or alias for Union[str, None]. It exists mostly as a convenience to help function signatures look a little cleaner`
So should I let `Optional["DataFrame"]` ?

Reply:
Oh, I see. Then shall we fix also `Series.sort_values` from `Union["Series", None]` to `Optional["Series"]` ??

Reply:
I guess so ! I fixed it